app/src/main/res/layout/comment_item.xml

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="?pixelBackgroundColor"
+    android:elevation="2dp">
+
+    <View
+        android:id="@+id/commentIndentation"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:background="?pixelSubtleBackground"
+        app:layout_constraintBottom_toBottomOf="@id/commentItemText"
+        app:layout_constraintEnd_toEndOf="@id/commentGuideline"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/commentGuideline"
+        android:layout_width="wrap_content"
+        android:layout_height="match_parent"
+        android:orientation="vertical"
+        app:layout_constraintGuide_percent="0" />
+
+    <View
+        android:id="@+id/commentIndentationBar"
+        android:layout_width="3dp"
+        android:layout_height="0dp"
+        android:background="?colorAccent"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintStart_toEndOf="@id/commentGuideline"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/commentItemAuthor"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:alpha="0.7"
+        android:paddingTop="10dp"
+        android:textSize="14sp"
+        app:layout_constraintStart_toStartOf="@id/commentItemText"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:text="momodao" />
+
+    <TextView
+        android:id="@+id/commentItemAuthorLabel"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="2dp"
+        android:alpha="0.7"
+        android:paddingTop="10dp"
+        android:text=" . "
+        android:textSize="14sp"
+        app:layout_constraintStart_toEndOf="@id/commentItemAuthor"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/commentItemPoint"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:alpha="0.7"
+        android:paddingTop="10dp"
+        android:textSize="14sp"
+        app:layout_constraintStart_toEndOf="@id/commentItemAuthorLabel"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:text="181" />
+
+    <TextView
+        android:id="@+id/commentItemPointLabel"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="2dp"
+        android:alpha="0.7"
+        android:paddingTop="10dp"
+        android:text="points . "
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/worker8/Pixels/blob/42c58c1774e85ce7400e4e3320af3e5ceffabd35/app/src/main/res/layout/comment_item.xml#L77"></span>Hardcoded string “points . “, should use <code>@string</code> resource</td>
    </tr>
  </tbody>
</table>